digits/templates/models/images/classification/new.html

```diff
@@ -340,7 +360,26 @@
                 <li><a href="#network-tab-custom" data-toggle="tab">Custom Network</a></li>
             </ul>
             <div class="tab-content well">
+                <div style="display:none;">{{form.framework(class='form-control')}}</div>
                 <div id="network-tab-standard" class="tab-pane active">
+                    <ul class="nav nav-tabs" id="stdnetRole">
+                        {% for fw in frameworks %}
+                        <li id="{{ fw.get_id() }}"><a href="#">{{ fw.get_name() }}</a></li>
```

Comment:
The `href="#"` doesn't really make sense here (and anchors to the top of the page if you click on it). I think you want a structure kinda like this:
```html
<div id="network-tab-standard" class="tab-pane active">
  <ul class="nav nav-tabs" id="stdnetRole">
    <li class="active"><a href="#standard-networks-caffe-tab">Caffe</a></li>
  </ul>
  <div id="standard-networks-caffe-tab" class="tab-pane active">
    Content here.
  </div>
</div>
```